xml/System.Formats.Tar/TarWriter.xml

```diff
@@ -113,19 +180,19 @@ The recommended format is <xref:System.Formats.Tar.TarFormat.Pax> for its flexib
       </Docs>
     </Member>
     <Member MemberName="Format">
-      <MemberSignature Language="C#" Value="public System.Formats.Tar.TarFormat Format { get; }" />
-      <MemberSignature Language="ILAsm" Value=".property instance valuetype System.Formats.Tar.TarFormat Format" />
+      <MemberSignature Language="C#" Value="public System.Formats.Tar.TarEntryFormat Format { get; }" />
+      <MemberSignature Language="ILAsm" Value=".property instance valuetype System.Formats.Tar.TarEntryFormat Format" />
       <MemberSignature Language="DocId" Value="P:System.Formats.Tar.TarWriter.Format" />
-      <MemberSignature Language="VB.NET" Value="Public ReadOnly Property Format As TarFormat" />
-      <MemberSignature Language="F#" Value="member this.Format : System.Formats.Tar.TarFormat" Usage="System.Formats.Tar.TarWriter.Format" />
-      <MemberSignature Language="C++ CLI" Value="public:&#xA; property System::Formats::Tar::TarFormat Format { System::Formats::Tar::TarFormat get(); };" />
+      <MemberSignature Language="VB.NET" Value="Public ReadOnly Property Format As TarEntryFormat" />
+      <MemberSignature Language="F#" Value="member this.Format : System.Formats.Tar.TarEntryFormat" Usage="System.Formats.Tar.TarWriter.Format" />
+      <MemberSignature Language="C++ CLI" Value="public:&#xA; property System::Formats::Tar::TarEntryFormat Format { System::Formats::Tar::TarEntryFormat get(); };" />
       <MemberType>Property</MemberType>
       <AssemblyInfo>
         <AssemblyName>System.Formats.Tar</AssemblyName>
         <AssemblyVersion>7.0.0.0</AssemblyVersion>
       </AssemblyInfo>
       <ReturnValue>
-        <ReturnType>System.Formats.Tar.TarFormat</ReturnType>
+        <ReturnType>System.Formats.Tar.TarEntryFormat</ReturnType>
       </ReturnValue>
       <Docs>
         <summary>The format of the archive.</summary>
```

Comment:
Can't provide suggestions beyond this line, but `TarFormat` needs to be changed to `TarEntryFormat` in this file in lines 233 and 240.